x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality_details/degraded_field_flyout/possible_mitigations/manual/pipeline_link.tsx

```diff
@@ -51,8 +55,8 @@ export function CreateEditPipelineLink({ isIntegration }: { isIntegration: boole
   const { type, name } = datasetDetails;
 
   const pipelineName = useMemo(
-    () => (isIntegration ? `${type}-${name}@custom` : `${type}@custom`),
-    [isIntegration, type, name]
+    () => (areIntegrationAssetsAvailable ? `${type}-${name}@custom` : `${type}@custom`),
```

Comment:
**nit**: not a big deal, but memoization that has to check 3 deps to for a ternary operator on strings seems a bit of a de-optimization.

Reply:
I can follow that argument. For a string there's also no benefit in trying to keep the referential identity stable.